cluster/validate-cluster.sh

```diff
@@ -52,7 +52,12 @@ CLUSTER_READY_ADDITIONAL_TIME_SECONDS="${CLUSTER_READY_ADDITIONAL_TIME_SECONDS:-
 
 EXPECTED_NUM_NODES="${NUM_NODES}"
 if [[ "${REGISTER_MASTER_KUBELET:-}" == "true" ]]; then
-  EXPECTED_NUM_NODES=$((EXPECTED_NUM_NODES+1))
+  if [[ "${KUBE_EXPERIMENTAL_REPLICATE_EXISTING_MASTER:-}" == "true" ]]; then
+    NUM_MASTERS=$(get-master-replicas-count)
```

Comment:
The idea is to check if all masters registered themselves? Maybe you could use this ConfigMap that you want to introduce to read this number?